apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.Tests/AggregationQueryResultsTest.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using Xunit;
+
+namespace Google.Cloud.Datastore.V1.Tests;
+
+public class AggregationQueryResultsTest
+{
+    [Fact]
+    public void Constructor()
```

Comment:
This should really test FromRunAggregationQueryResponse, at which point the constructor itself can be made private.
```suggestion
    public void Constructor()
```